CHANGELOG.rst

```diff
@@ -2,6 +2,211 @@
 Release Notes
 =============
 
+.. _Release Notes_7.61.0:
+
+7.61.0
+======
+
+.. _Release Notes_7.61.0_Prelude:
+
+Prelude
+-------
+
+Release on: 2025-01-13
+
+- Please refer to the `7.61.0 tag on integrations-core <https://github.com/DataDog/integrations-core/blob/master/AGENT_CHANGELOG.md#datadog-agent-version-7610>`_ for the list of changes on the Core Checks
+
+
+.. _Release Notes_7.61.0_Upgrade Notes:
+
+Upgrade Notes
+-------------
+
+- Upgraded JMXFetch to `0.49.6 <https://github.com/DataDog/jmxfetch/releases/0.49.6>`_ which fixes a ``NullPointerException`` on
+  JBoss when user and password not set. See `0.49.6 <https://github.com/DataDog/jmxfetch/releases/tag/0.49.6>`_ for more details.
+
+- Windows containers were updated to use OpenJDK 11.0.25+9.
+
+
+.. _Release Notes_7.61.0_New Features:
+
+New Features
+------------
+
+- Add metrics origins for Nvidia Nim integration.
+
+- APM: New configuration apm_config.obfuscation.credit_cards.keep_values (DD_APM_OBFUSCATION_CREDIT_CARDS_KEEP_VALUES)
+  can be used to skip specific tag keys that are known to never contain credit card numbers. This is especially useful
+  in cases where a span tag value is a number that triggers false positives from the credit card obfuscator.
+
+- Add new metric, ``container.restarts``, which indicates the number of times a container has been restarted due to the restart policy.
+  For more details: https://docs.docker.com/engine/containers/start-containers-automatically/.
+
+- APM: Introducing the Error Tracking Standalone config option. Only span chunks
+  that contain errors or exception OpenTelemetry span events are taken into
+  consideration by sampling.
+
+- Add new windows images for LTSC 2019 and LTSC 2022:
+  - `datadog-agent:7-servercore-ltsc2019-amd64`
+  - `datadog-agent:7-servercore-ltsc2022-amd64`
+  - `datadog-agent:7-servercore-ltsc2019-jmx-amd64`
+  - `datadog-agent:7-servercore-ltsc2022-jmx-amd64`
+  - `datadog-agent:latest-servercore-ltsc2019-jmx`
+  - `datadog-agent:latest-servercore-ltsc2022-jmx`
+  - `datadog-agent:latest-servercore-ltsc2019`
+  - `datadog-agent:latest-servercore-ltsc2022`
+  - `datadog-agent:7.X.Y-ltsc2019`
+  - `datadog-agent:7.X.Y-ltsc2022`
+  - `datadog-agent:7.X.Y-ltsc2019-jmx`
+  - `datadog-agent:7.X.Y-ltsc2022-jmx`
+  - `datadog-agent:7.X.Y-servercore-ltsc2019`
+  - `datadog-agent:7.X.Y-servercore-ltsc2022`
+  - `datadog-agent:7.X.Y-servercore-ltsc2019-jmx`
+  - `datadog-agent:7.X.Y-servercore-ltsc2022-jmx`
+  - `datadog-agent:latest-ltsc2019`
+  - `datadog-agent:latest-ltsc2022`
+
+- [ha-agent] Add haagent component used for HA Agent feature.
+
+- The cluster-agent now can collect pod disruption budgets from the cluster.
```

Comment:
Should this entry be moved to `CHANGELOG-DCA.rst` instead ?

Reply:
Good catch, fixed!